modules/core/include/opencv2/core/hal/intrin_rvv.hpp

```diff
@@ -2721,8 +2601,7 @@ inline v_int64x2 v_dotprod_fast(const v_int32x4& a, const v_int32x4& b, const v_
 inline v_uint32x4 v_dotprod_expand_fast(const v_uint8x16& a, const v_uint8x16& b)
 {
     unsigned CV_DECL_ALIGNED(32) ptr[16] = {0};
-    vsetvlmax_e32m4();
-    vse32_v_u32m4(ptr, vqmaccu_vv_u32m4(vzero_u32m4(), a, b));
+    vse32_v_u32m4(ptr, vwcvtu_x_x_v_u32m4(vwmulu_vv_u16m2(a, b, 16), 16), 16);
```

Comment:
Why not presume `vqmacc` intrinsic instead of `vwcvtu` & `vwmulu`?

Reply:
Hi Alex. Thanks for the review! In the latest V extension specification,` vq*` prefix instructions are not included, which is regarded as a possible addition in a future extension. Therefore, in the latest riscv-gnu-toolchain, these instructions are also removed. So I replaced it with the implementation above.

Reply:
Got it. Thanks for clarification.